.buildkite/pipeline.yml

```diff
@@ -325,189 +325,146 @@ steps:
-          --FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false
-          --turbconv_case Soares
-          --z_elem 150 --z_stretch false --z_max 3750
-          --t_end 8hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins
-          --moist equil
+          --regression_test true
+          --FLOAT_TYPE Float64
           --job_id edmf_soares
           --dt 0.25secs
+          --toml ./.buildkite/toml/soares.toml
         artifact_paths: "edmf_soares/*"
         agents:
           slurm_mem: 20GB
-
       - label: ":balloon: SCM EDMF Soares (Const entr, Pi detr)"
         command: >
           julia --color=yes --project=examples examples/hybrid/driver.jl
-          --config column --turbconv edmf --debugging_tc true
-          --FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false
-          --turbconv_case Soares
-          --z_elem 150 --z_stretch false --z_max 3750
-          --t_end 8hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins
-          --moist equil
+          --FLOAT_TYPE Float64
           --edmf_entr_closure PiDetrainment
           --job_id edmf_soares_const_entr_pi_detr
           --quicklook_reference_job_id edmf_soares
           --dt 0.5secs
+          --toml ./.buildkite/toml/soares.toml
         artifact_paths: "edmf_soares_const_entr_pi_detr/*"
         agents:
           slurm_mem: 20GB
 
       - label: ":balloon: SCM EDMF Soares (Const entr, pi detr, JFNK IMEX EDMF)"
         command: >
           julia --color=yes --project=examples examples/hybrid/driver.jl
-          --config column --turbconv edmf --debugging_tc true
-          --FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false
-          --turbconv_case Soares
-          --z_elem 150 --z_stretch false --z_max 3750
-          --t_end 8hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins
-          --moist equil
+          --FLOAT_TYPE Float64
           --edmf_entr_closure PiDetrainment
           --imex_edmf_turbconv true
           --imex_edmf_gm true
           --use_krylov_method true
           --quicklook_reference_job_id edmf_soares
           --job_id edmf_soares_const_entr_jfnk_imex
           --dt 50secs
+          --toml ./.buildkite/toml/soares.toml
         artifact_paths: "edmf_soares_const_entr_jfnk_imex/*"
         soft_fail: true
         agents:
           slurm_mem: 20GB
-
       - label: ":balloon: SCM EDMF Bomex (Default: compressible, Newton's method)"
         command: >
           julia --color=yes --project=examples examples/hybrid/driver.jl
-          --config column --turbconv edmf --debugging_tc true --regression_test true
-          --FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false
-          --turbconv_case Bomex --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex
-          --z_elem 60 --z_stretch false --z_max 3e3
-          --t_end 6hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins
-          --moist equil
```

Comment:
Should the options that are moved to the `toml` files be deleted from the supported `cli_args`? Otherwise we are running into issues who overwrites whom.

Reply:
I think they should still be supported for now, since people probably rely on them for their workflow. The specified CLI arguments always take priority over the TOML file. I can add a note in the documentation to clarify this?

Reply:
Sounds good. Adding it to the docs would be great. Not sure where it would be most useful. Maybe in the `cli_args` docstring for the `--toml` option?

Reply:
I'll at least add to the docstring in `cli_options.jl`. I can look around if there is another place in the actual docs as well.